vnext/src/Libraries/core/setUpDeveloperTools.windows.js

```diff
@@ -75,8 +75,6 @@ if (__DEV__) {
       });
     }
 
-    // [Windows
-    // require('./setUpReactRefresh');
-    // Windows]
```

Comment:
It looks like we don't need this forked file anymore.

Reply:
I can remove this in the 61 working branch since we needed to touch here anyway.